architecture/core_objects/images_containers.adoc

```diff
@@ -0,0 +1,72 @@
+= Overview
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+
+== Containers
+
+The basic units of OpenShift applications are called containers. Linux
+container technologies are lightweight mechanisms for isolating
+running processes so that they are limited to interacting with only
+their designated resources. Many application instances can be running
+in containers on a single host without visibility into each others'
+processes, files, network, and so on. Typically we expect that each will
+provide a single service (often called a "micro-service"), such as a
+web server or a database, though they can be used for arbitrary workloads.
```

Comment:
each = ambiguous. Also, "we" might have to be avoided.

Suggest: "Typically, each container provides a single service..."